cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -34,6 +34,12 @@ data:
   # Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.
   # CSI_LOG_LEVEL: "0"
 
+  # Set the number of replicas of rbdplugin-provisioner
+  # CSI_RBD_PLUGIN_PROVISIONER_REPLICAS: "2"
```

Comment:
can we just check the number of worker nodes in code and set replica to 1?
With this approach, a user can set a higher value and it creates more replicas?

Reply:
It's a nice idea.

I have one question. Do you recommend not to the number of replicas larger than 2? If so, I just modify this PR as you suggested. Otherwise, it's also nice to make it configurable and the default value becomes 1 or 2 depending on the number of nodes.

Reply:
Yes  earlier the count was 3. as it consumes more resources and provisioner will always be in HA, what we thought was 2 is good enough

Reply:
OK, I'll fix it.